### ISSUE TITLE
Tanks are often unable to fire their cannons

### ISSUE THREAD

**Issue by cmbasnett** [DarklightGames/DarkestHour#558]
Often (in multi-player), tanks are suddenly unable to fire their tank cannons. Changing positions and then back to the tank cannon allows them to fire. Also affects anti-tank weapons like the Flak 88. This is a critical bug and needs to be fixed as soon as possible.

**Comment by handsm** [comment]
Probable/possible fix in commit mentioned above.

Certainly a fix for the only way we have been able to recreate this bug: using the manual cannon reloading option and switching seats after firing. I have confirmed that is fixed, not just by the bug failing to appear, but by actively logging that a bug would have occurred under the old code line, but a reload has proceeded under the new.

If there is still a problem with cannons not being able to fire using automatic reloading option - which we have been unable to recreate - then that is something separate. But given that the manual reload problem has been reproducable, and that problem has been identified and fixed, there is a good chance this nails it. Leaving the issue open for now though - too early to count this as done.
